src/deploy.ps1

```diff
@@ -82,5 +91,31 @@ function PublishToBlobStorage {
 	Set-AzureStorageBlobContent -File $targetDllPath -Container $BlobStorageContainer -Blob $blobName -Context $blobContext -Force
 }
 
+
+function UploadAssembliesToAdls() {
+	# Check to see whether we need to upload to blob
+	if([string]::IsNullOrEmpty($AzureDataLakeStoreName)) {
+        Write-Host "No data lake information specified. Not uploading to storage."
+		return
+	}
+
+	Login-AzureRmAccount
+	Set-AzureRMContext -SubscriptionId $SubscriptionId 
```

Comment:
Yeah, I figured this. It was just temporary and I was going to raise another issue. I had tried non-interactive with an SP likeso:

Login-AzureRmAccount -ServicePrincipal -ApplicationId  xx -Credential $pscredential -TenantId $tenantid

But haven't quite got it working it.